wagtail/admin/rich_text/converters/contentstate_models.py

```diff
@@ -0,0 +1,88 @@
+import json
+import random
+import string
+
+
+class Block(object):
+    def __init__(self, typ, depth=0):
+        self.type = typ
+        self.depth = depth
+        self.text = ""
+        self.key = ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(5))
+        self.inline_style_ranges = []
+        self.entity_ranges = []
```

Comment:
I think this is fine as it is - the classes here aren't really intended to directly mirror Draft.js's implementation, only to provide friendlier datatypes for contentstate.py to work with (rather than fiddling with dicts directly).